src/NUnitFramework/tests/nunit.framework.tests-4.5.csproj

```diff
@@ -207,6 +207,7 @@
     <Compile Include="Constraints\ToStringTests.cs" />
     <Compile Include="Constraints\UniqueItemsConstraintTests.cs" />
     <Compile Include="Constraints\XmlSerializableTest.cs" />
+    <Compile Include="Interfaces\TestOutputTests.cs" />
```

Comment:
I like it! But now you have to add it to the other five projects. :-) 

That's 2.0, 3.5, 4.0 and netstandard 1.3 and 1.6.

Reply:
Is there a specific way to add them? Or simply copy/paste? I noticed that files in different projects are linked somehow: changes in file propagate to others

Reply:
You just add them. The separate csproj files are all located in the same directory so no link is needed. It's tedious but easy. 😄

Reply:
I see now. Thanks @CharliePoole

Reply:
This is about to get much easier really soon when we move to the new csproj SDK which doesn't require listing `<Compile>` items in the csproj at all.